src/Akka.Persistence.Redis.Cluster.Tests/RedisJournalDefaultDatabaseSpec.cs

```diff
@@ -0,0 +1,99 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="RedisJournalSpec.cs" company="Akka.NET Project">
+//      Copyright (C) 2013-2021 .NET Foundation <https://github.com/akkadotnet/akka.net>
+// </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using Akka.Configuration;
+using Akka.Persistence.TCK.Journal;
+using FluentAssertions;
+using StackExchange.Redis;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Akka.Persistence.Redis.Cluster.Tests
+{
+    [Collection("RedisClusterSpec")]
+    public class RedisJournalDefaultDatabaseSpec : JournalSpec
+    {
+        private readonly RedisClusterFixture _fixture;
+
+        public static Config Config(RedisClusterFixture fixture)
+        {
+            DbUtils.Initialize(fixture);
+
+            return ConfigurationFactory.ParseString($@"
+            akka.loglevel = INFO
+            akka.persistence.journal.plugin = ""akka.persistence.journal.redis""
+            akka.persistence.journal.redis {{
+                class = ""Akka.Persistence.Redis.Journal.RedisJournal, Akka.Persistence.Redis""
+                plugin-dispatcher = ""akka.actor.default-dispatcher""
+                configuration-string = ""{DbUtils.ConnectionString},defaultDatabase=0""
```

Comment:
Set this to a non-zero value to ensure that it's picked.

Reply:
And then validate that we're talking to the non-zero database.

Reply:
And is that a function of how we're using the driver?

Reply:
Or, should we support multiple servers? I thought cluster mode took care of the "scale out" use case already.

Reply:
It appears we will have to setup multiple servers explicitly. Right now it is only one server - `0`!

Reply:
Multiple databases not supported with redis cluster according to this comment https://github.com/StackExchange/StackExchange.Redis/issues/1188#issuecomment-513149394